storybook/src/components/Select/Select.docs.mdx

```diff
@@ -0,0 +1,36 @@
+import { Canvas, Controls, Markdown, Meta, Primary } from "@storybook/blocks";
+import * as SelectStories from "./Select.stories.tsx";
+import README from "../../../../packages/css/src/components/select/README.md?raw";
+
+<Meta of={SelectStories} />
+
+<Markdown>{README}</Markdown>
+
+# Default
+
+<Primary />
+
+<Controls />
+
+## Multiple
+
+Avoid adding functionality to allow selecting multiple options. Multi select is harder to use
+on desktop as they require the user to hold down the `Ctrl` or `Cmd` key while clicking on the options.
+It is recommended to use checkboxes instead.
+
+- [WCAG 2.2 success criterion](https://www.w3.org/WAI/WCAG22/Understanding/dragging-movements.html)
```

Comment:
Is this relevant to multi select interaction?

Reply:
Yes this is the gov.uk reference to why you should not allow multiple. Since we don't actively prevent the usage (you can still add the attribute and the styling is there) I included this like for the curious developer.